assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -162,33 +163,29 @@ export const TakeActionBoxoutEditor = ({
             label={__('Select Take Action Page:', 'planet4-blocks-backend')}
             value={take_action_page}
             options={[
-              { label: __('None', 'planet4-blocks-backend'), value: 0 },
+              { label: __('None (custom)', 'planet4-blocks-backend'), value: 0 },
```

Comment:
I'll check this with design, but it seems more sensible to show specifically what this option does, more than just not having an act page selected. That way we can also hide the customize label entirely if an act page is selected, which further makes the block less confusing.